modules/ROOT/pages/queries/select-version.adoc

```diff
@@ -0,0 +1,251 @@
+:description: Information about how to select Cypher version for queries or databases.
+
+= Select Cypher version
+
+Users can specify the version of Cypher in which they want to run their queries, choosing between Cypher 5 and Cypher 25.
+The Cypher version can be set for individual queries or as the default language for a database.
+
+[[cypher-versions-explained]]
+== Cypher 5, Cypher 25, and Neo4j explained
+
+Starting in 2025, the Neo4j server transitioned to a calendar-based versioning system.
+This means Neo4j will no longer use its previous semantic versioning and release pattern (e.g., 5.25, 5.26).
+Instead, releases from 2025 onwards will follow a *YYYY.MM* format, beginning with version 2025.01 released in January 2025, followed by 2025.02 released in February 2025, and so on.
+
+Cypher 25 is introduced alongside Neo4j 2025.03 and is the default language for newly created databases running version 2025.03 or later.
+However, users running Neo4j version 2025.03 or later can choose to run their queries using the previous version of Cypher: Cypher 5.
+If so, Neo4j will use Cypher as it existed at the time of the Neo4j 5.26 long-term support release.
```

Comment:
I think this is the first place we mention `Neo4j 5.26 long-term support release` but then it's first on line 123 we do `the 5.26 long-term support (LTS) release`, feels like something we could do the first time we mention the release and then use throughout?

also on line 123 we don't mention Neo4j so it's not that consistent either :( might be good to go over and check all mentions of 5.26 to see that they are consistent?

Reply:
I also think line 123 is the only one that says `(LTS)` whereas all other mentions just say `long-term support release` 🤷

Reply:
Will standardize 👍